tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -251,6 +252,19 @@ async def set_permission(self, descriptor, state, origin):
         return await self.webdriver.bidi_session.permissions.set_permission(
             descriptor=descriptor, state=state, origin=origin)
 
+class WebDriverBidiWebExtensionsProtocolPart(WebExtensionsProtocolPart):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self.webdriver = None
+
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+
+    async def load_web_extension(self, path):
+        return await self.webdriver.bidi_session.web_extension.install(path)
```

Comment:
should be 
`return self.webdriver.loop.run_until_complete(self.webdriver.bidi_session.web_extension.install(path))`